save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -46,17 +47,16 @@ class UserSettingsOrganizationsMenuView : UserSettingsView() {
                         div {
                             className = ClassName("row justify-content-between align-items-center")
                             div {
-                                className = ClassName("align-items-center ml-3")
+                                className = ClassName("align-items-center ml-3 text-primary")
```

Comment:
logic for `state.selfOrganizationWithUserList`, `state.selfDeletedOrganizationWithUserList` and `state.selfBannedOrganizationWithUserList` looks similar. Please combine it with different by `organizationWithUsers.organization.status` or extract common places to separate methods